mxnet/inference/docker/1.9/py3/Dockerfile.graviton.ec2.cpu

```diff
@@ -20,6 +20,7 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
     TEMP=/home/model-server/tmp
 
 RUN apt-get update \
+ && apt-get -y upgrade \
```

Comment:
Lets remove this completely? It will be better to let this be handled in the future. In case we make this change today but do not release, it might become complicated to trace any bug or issue in the future.

Reply:
this is required for get past the sanity check for another vul-id on the OS. The current frameworks already had it.

Reply:
Missed, this is MXnet.. doesn't matter here anyway. as I believe this one is deprecated